Content.Client/GameObjects/Components/Damage/DamageableComponent.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Content.Shared.GameObjects.Components.Damage;
+using Robust.Shared.GameObjects;
+
+namespace Content.Client.GameObjects.Components.Damage
+{
+    [RegisterComponent]
+    [ComponentReference(typeof(IDamageableComponent))]
+    [ComponentReference(typeof(SharedDamageableComponent))]
+    public class DamageableComponent : SharedDamageableComponent
```

Comment:
Can't this be deleted if it doesn't do anything differently from shared?

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/2783/commits/81ea2bfe805ee489a797b953f7506b711e30834e